packages/x-tree-view/src/internals/corePlugins/useTreeViewId/useTreeViewId.ts

```diff
@@ -37,4 +37,4 @@ useTreeViewId.params = {
   id: true,
 };
 
-useTreeViewId.getInitialState = ({ id }) => ({ id: { treeId: id ?? createTreeViewDefaultId() } });
+useTreeViewId.getInitialState = () => ({ id: { treeId: undefined } });
```

Comment:
Couldn't we use `id` in the `getInitialState` if it is provided? 🤔 
```suggestion
useTreeViewId.getInitialState = ({ id }) => ({ id: { treeId: id ?? undefined } });
```

Reply:
I'll check if it causes problem :+1:

Reply:
Updated